packages/shared-component--membershipHero/contentful/membershipHero.js

```diff
@@ -0,0 +1,150 @@
+module.exports = function (migration) {
+    const hero = migration.createContentType('membershipHero');
+
+    hero.name('-- Membership hero')
+        .description('A page header that can be used to promote membership.')
+
+    hero.createField('name')
+        .name('Name')
+        .type('Symbol')
+        .required(true)
+
+    hero.createField('homeHero')
+        .name('Main homepage hero?')
+        .type('Boolean')
+
+    hero.createField('heroBackground')
+        .name('Background colour')
+        .type('Object')
+
+    hero.createField('heading')
+        .name('Heading')
+        .type('Symbol')
+        .required(true)
+        .validations([
+            {
+                size: { max: 90 },
+                message: "Text in this field must be less than 90 characters."
+            }
+        ]);
+
+    hero.createField('bodyText')
+        .name('Body text')
+        .type('Text')
+        .required(true)
+        .validations([
+            {
+                size: { max: 170 },
+                message: "Text in this field must be less than 170 characters."
+            }
+        ]);
+
+    hero.createField('appStoreLink')
+        .name('App store link')
+        .type('Link')
+        .linkType('Entry')
+        .validations([
+            {
+                linkContentType: ['externalLink', 'internalLink']
+            }
+        ]);
+
+    hero.createField('googlePlayLink')
+        .name('Google play link')
+        .type('Link')
+        .linkType('Entry')
+        .validations([
+            {
+                linkContentType: ['externalLink', 'internalLink']
+            }
+        ]);
+
+    hero.createField('button')
+        .name('Button')
+        .type('Link')
+        .linkType('Entry')
+        .validations([
+            {
+                linkContentType: ['externalLink', 'internalLink']
+            }
+        ]);
+
+    hero.createField('link')
+        .name('Additional link')
+        .type('Link')
+        .linkType('Entry')
+        .validations([
+            {
+                linkContentType: ['externalLink', 'internalLink']
+            }
+        ]);
+
+    hero.createField('image')
+        .name("Image")
+        .type("Link")
+        .linkType("Asset")
+        .validations([
+            {
+                linkMimetypeGroup: ["image"]
+            }
+        ]);
+
+    hero.createField('imageType')
+        .name('Type of hero image')
+        .type('Symbol')
+        .validations([
+            {
+                in: ['Home', 'Handset', 'Photo']
```

Comment:
There's only a 'Handset' or 'Photo' image type, isn't there?

We're going to use the **Main homepage hero?** boolean for the font size modifier.

Reply:
updated the script to remove `home`